src/Java/Visualization/MultipleViewports.java

```diff
@@ -0,0 +1,69 @@
+import vtk.vtkActor;
+import vtk.vtkNativeLibrary;
+import vtk.vtkPolyDataMapper;
+import vtk.vtkRenderWindow;
+import vtk.vtkRenderWindowInteractor;
+import vtk.vtkRenderer;
+import vtk.vtkSphereSource;
+
+public class MultipleViewports 
+{
+  // -----------------------------------------------------------------
+  // Load VTK library and print which library was not properly loaded
+  static 
+  {
+    if (!vtkNativeLibrary.LoadAllNativeLibraries()) 
+    {
+      for (vtkNativeLibrary lib : vtkNativeLibrary.values()) 
+      {
+        if (!lib.IsLoaded()) 
+        {
+          System.out.println(lib.GetLibraryName() + " not loaded");
+        }
+      }
+    }
+    vtkNativeLibrary.DisableOutputWindow(null);
+  }
+  // -----------------------------------------------------------------
+
+  public static void main(String args[]) 
+  {
+        
+    //  Define viewport ranges
+    double xmins[] = new double[] {0,.5,0,.5};
+    double xmaxs[] = new double[] {0.5,1,0.5,1};
+    double ymins[] = new double[] {0,0,.5,.5};
+    double ymaxs[] = new double[] {0.5,0.5,1,1};
```

Comment:
Check formatting - use a space after comma and instead of .5 use 0.5 etc.

Reply:
formatted. Please review.